policyengine_us/parameters/gov/irs/deductions/itemized/salt_and_real_estate/state_sales_tax.yaml

```diff
@@ -0,0 +1,5696 @@
+description: Filers electing to itemize for state tax or other purposes follow this sales tax.
+metadata:
+  unit: currency-USD
+  period: year
+  label: Optional State Sales Rate
+  reference:
+    - title: 2023 Optional State Sales Tax
+      href: https://www.irs.gov/instructions/i1040sca#en_US_2023_publink10005349
+
+Alabama:
+  1:
+    0: 365
+    20_000: 490
+    30_000: 551
+    40_000: 602
+    50_000: 646
+    60_000: 685
```

Comment:
```suggestion
    1: 365
    2: 490
    3: 551
    4: 602
    5: 646
    6: 685
```

Reply:
We will need to denominate those as integers and then create a ENUM variable which determines that 
0-$20,000 = 1
$20,000 - $40,000 = 2
etc...